src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2547,7 +2547,7 @@ static void Main(string[] args)
             var tree = compilation.SyntaxTrees.Single();
             var model = compilation.GetSemanticModel(tree);
 
-            var expr = tree.GetRoot().DescendantNodes().OfType<ParameterSyntax>().Single();
+            var expr = tree.GetRoot().DescendantNodes().OfType<ParameterSyntax>().First();
```

Comment:
it is related.  We now parse hte above code much better than before.  This means we now see multiple parameter (the "string[] args" one and the "x" in "x =>" one).  So "Single" was causing this test to crash.